### PR TITLE
Remove general tab in parameters

### PR DESCRIPTION
**Pull request by etiennehomer**
Signed-off-by: HOMER Etienne <etiennehomer@gmail.com>

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -236,7 +237,8 @@ const App = () => {
                                 <GridActionsLogoDark />
                             )
                         }
-                        onParametersClick={() => showParametersClicked()}
+                        //To uncomment when parameters are added to gridactions
+                        // onParametersClick={() => showParametersClicked()}
```
Comment (FranckLecuyer) [inline src/components/app.js]: Maybe, the 'Parameters' item should be disabled, because there are no parameters yet.
And, we cannot change theme anymore

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/app.js (hunk 1 above)
Comment (ChamseddineBhd) [reply src/components/app.js]: i agree with Franck we should disable parameters item.

**Review by ChamseddineBhd — COMMENTED** [review]
I think we should remove Parameters component and delete commented functions as we don't have parameters for this app. we should avoid unused components and functions in the code !
parameters in the menu should  be disabled (or maybe remove the option if callback is undefined later)
what do you think @jonenst

File: src/components/app.js (hunk 1 above)
Comment (etiennehomer) [reply src/components/app.js]: That's the behavior when we don't give the call back. As the callback is undefined, parameters button will be disabled in commons ui.

Comment (etiennehomer) [reply src/components/app.js]: Commented code has been removed.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "private": true,
     "dependencies": {
         "notistack": "^1.0.1",
-        "@gridsuite/commons-ui": "0.7.0",
+        "@gridsuite/commons-ui": "0.8.0",
```
Comment (ChamseddineBhd) [inline package.json]: to do in a separate PR

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (etiennehomer) [reply package.json]: Ok

**Review by ChamseddineBhd — APPROVED** [review]

**Review by FranckLecuyer — APPROVED** [review]
